tests/acceptance/features/bootstrap/Sharing.php

```diff
@@ -3819,8 +3819,7 @@ public function getLastPublicSharePath():string {
 	 */
 	public function getPublicPreviewOfFile(string $fileName, string $token):void {
 		$url = $this->getBaseUrl() .
-			"/index.php/apps/files_sharing/ajax/publicpreview.php" .
-			"?file=$fileName&t=$token";
+			"/remote.php/dav/public-files/$token/$fileName?preview=1";
```

Comment:
@amrita-shrestha Will this endpoint works as well? I think `remote.php` will be gone soon (but not sure)
```suggestion
			"/dav/public-files/$token/$fileName?preview=1";
```

Reply:
Actually, it should not work without remote.php but some endpoint work without remote.php

Reply:
:thinking: